### PR TITLE
Updated Installation Instructions +

### PR DESCRIPTION
**Pull request by JohnBaku**
A couple of changes after going through installation process myself:

- Updated installation instructions
- Separated Bugs and Contributing
- Added Features and Requirements
- Updated Screenshot of apps

### CODE REVIEW

**Comment by JohnBaku** [comment]
v1

File: README.md [hunk 1]
```diff
@@ -1,43 +1,65 @@
...
-- Build the `FetLife` scheme in Xcode. The app can be run within a simulator or [on a device](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/LaunchingYourApponDevices/LaunchingYourApponDevices.html).
 
-If you find a bug or want to contribute, please start by reading through the [GitHub issues](https://github.com/fetlife/fetlife-ios/issues) and [contributing](https://github.com/fetlife/fetlife-ios/blob/master/CONTRIBUTING.md) guidelines.
+### Installing the App on your iPhone for the first time
 
-## Screenshots
+These instructions are written assuming you know very little about computers and to help get the app on your iPhone as quickly as possible:
 
-![](https://cloud.githubusercontent.com/assets/171215/14657747/32b8daf6-0655-11e6-9629-11df51b569b7.png)
+1. Install the latest version of [Xxode](https://itunes.apple.com/ca/app/xcode/id497799835?mt=12).
```
Comment (jacortinas) [inline README.md]: Xxode -> Xcode

File: README.md [hunk 2]
```diff
@@ -1,43 +1,65 @@
...
 
-The app is still a major work in progress. The goal is to take the base FetLife chat application and add more features and API endpoints to match the other features on fetlife.com.
+- View a list of your conversations
+- Read and respond to conversations
+- Receive a push notification when you receive a new message
 
-The current operating system target of the application is iOS 9.0 or higher.
 
-#### Building the application
+### Requirements to run app on your iPhone
```
Comment (philippt) [inline README.md]: to run *the* app?

File: README.md [hunk 3]
```diff
@@ -1,43 +1,65 @@
...
-1. Download and install the latest [Xcode developer tools](https://developer.apple.com/xcode/download/) from Apple.
-- Install [CocoaPods](https://cocoapods.org/).
-- Clone the [repository](https://github.com/fetlife/fetlife-ios).
 
-  ```
-  git clone git@github.com:fetlife/fetlife-ios.git
-  cd fetlife-ios
-  ```
-- Install the project's dependencies.
+### Screenshots of App
```
Comment (philippt) [inline README.md]: ...same here - I'd either make it "Screenshots" or "of *the* App"

File: README.md [hunk 4]
```diff
@@ -1,43 +1,65 @@
...
+  - "Select a Development Team" i.e your own name.
+  - *You will probably have to do that 7-8 times*
+  - Unlock your phone so your Mac can install the app onto your phone.
+6. Do the [happy dance](https://www.youtube.com/watch?v=Ckt5JgshnaA)! :-)
 
-## License
+
+### Kinksters Helping Kinksters
+
+Want to install the app on your phone but not technically savvy? Ask your local kinky geek! Technically savvy and want to give back to the community... bring your laptop with you the next time you attend an event and install / update the app for anyone who's interested in having it install on their iPhone. *#KinkstersHelpingKinksters*
```
Comment (philippt) [inline README.md]: Might be just me, but I'd expect "but *are* not technically savvy?"

File: README.md [hunk 5]
```diff
@@ -1,43 +1,65 @@
...
+
+
+### Got Bugs?
+
+If you find a bug please start by reading through the our current list of [open issues](https://github.com/fetlife/fetlife-ios/issues) and if you can't find anything about your bug please [submit a new bug]().
+
+
+### Want to Contribute
+
+Woot woot! Please checkout our [Contributing Guidelines](https://github.com/fetlife/fetlife-ios/blob/master/CONTRIBUTING.md) and go from their.
```
Comment (philippt) [inline README.md]: from *there*

**Comment by philippt** [comment]
:+1:

File: README.md (hunk 3 above)
Comment (JohnBaku) [inline README.md]: @philippt your english is at least 10x better then mine... damn! :-/ Thanks!!!!

File: README.md (hunk 4 above)
Comment (JohnBaku) [inline README.md]: @philippt good catch! Fixing.

File: README.md [hunk 6]
```diff
@@ -1,43 +1,64 @@
...
-## License
+
+### Kinksters Helping Kinksters
+
+Want to install the app on your phone but are not technically savvy? Ask your local kinky geek! Technically savvy and want to give back to the community... bring your laptop with you the next time you attend an event and install / update the app for anyone who's interested in having it install on their iPhone. *#KinkstersHelpingKinksters*
+
+
+### Got Bugs?
+
+If you find a bug please start by reading through the our current list of [open issues](https://github.com/fetlife/fetlife-ios/issues) and if you can't find anything about your bug please [submit a new bug]().
```
Comment (jacortinas) [inline README.md]: Is there supposed to be a link in the `[submit a new bug]()` markdown?

**Comment by jacortinas** [comment]
:+1:

**Pull request merged**